### DISCUSSION TITLE
Problem with paging feed versions API endpoint

### DISCUSSION THREAD

**Discussion by alenastern** [transitland/transitland#361] [Q&A] [answered]
Hello! I've been paging the feed versions v2 API endpoint to identify feed versions that correspond with a specific target date range. Code that worked successfully to paginate a couple of months ago is now no longer working. For example, when I query the following URL, I successfully get results (api key hidden throughout for privacy):

`r <- GET("https://transit.land/api/v2/rest/feeds/f-dr5r-mtanewyorkcitytransit/feed_versions?api_key={api_key}&limit=100")`

and when I examine the meta field in the response, I see that there is a next URL provided:

```
$after
[1] 271351

$`next`
[1] "https://api.transit.land/api/v2/rest/feeds/f-dr5r-mtanewyorkcitytransit/feed_versions?after=271351&api_key={api_key}&limit=100"
```

But when I then run the following code, the response has a 200 status code, but the content of the response is empty:
```
> r <- GET("https://api.transit.land/api/v2/rest/feeds/f-dr5r-mtanewyorkcitytransit/feed_versions?after=271351&api_key={api_key}&limit=100")
> status_code(r)
[1] 200
> content <- content(r)
> content
NULL
```

I've tried this for a few different operators and have gotten the same results. The behavior was also consistent when not setting the limit parameter in the GET request. 

I saw the [similar issue](https://github.com/transitland/transitland/discussions/358) with the feeds endpoint, so I'm wondering if this is perhaps a result of the same bug that didn't get resolved in the fix for that issue? Thank you for your help!

**Comment by irees** [comment, marked as answer]
Hi @alenastern - last week there was some significant change behind the scenes on how pagination works; perhaps unsurprising there are a few wrinkles to iron out. I'm investigating this and should have a resolution soon.

**Reply by alenastern** [reply]
Thanks for your quick reply @irees! I appreciate you looking into this issue.

**Reply by irees** [reply]
@alenastern We've deployed a fix; can you try again and see if the issue is resolved?

**Reply by alenastern** [reply]
@irees I just ran a couple of tests and it looks good! Thanks so much for the quick fix and I'll follow up if I encounter any other issues!
